doc/nrf/ug_thread_architectures.rst

```diff
@@ -111,13 +110,9 @@ This platform design is suitable for the following development kits:
 |:ref:`nRF52833 DK <ug_nrf52>`   |PCA10010   |:ref:`nrf52833dk_nrf52833 <nrf52833dk_nrf52833>`|``nrf52833dk_nrf52833``        |
 +--------------------------------+-----------+------------------------------------------------+-------------------------------+
 |:ref:`nRF5340 DK <ug_nrf5340>`  |PCA10095   |:ref:`nrf5340dk_nrf5340 <nrf5340dk_nrf5340>`    |``nrf5340dk_nrf5340_cpuapp``   |
-|                                |           |                                                |                               |
```

Comment:
No, the two targets should be displayed in separate lines, so this is actually wrong in line 73 (and correct here).